src/bsoncxx/json.cpp

```diff
@@ -274,7 +274,12 @@ std::string to_json(types::value value) {
 
     json_visitor v(ss, false, 0);
 
-    switch ((int)value.type()) {
+    if (!value.type()) {
+        // TODO: use bsoncxx error category
+        throw std::runtime_error("invalid value");
```

Comment:
how would a view (containing pre-serialized bson) contain invalid elements? that would seem to indicate a bug in the library itself